templates/taxi/manufacturer_confirm_delete.html

```diff
@@ -0,0 +1,11 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Delete Manufacturer</h1>
+  <p>Do you really want to delete manufacturer - {{ manufacturer.name }}?</p>
+  <form action="" method="post">
+    {% csrf_token %}
+    <input type="submit" value="Yes" class="btn btn-danger">
+    <a class="btn btn-primary" href="{% url "taxi:manufacturer-list" %}">NO</a>

```

Comment:
The 'Cancel' button should redirect to the previous page instead of the manufacturer list page. You can achieve this by using JavaScript to go back in history, such as `onclick="window.history.back(); return false;"`, or by setting the appropriate URL dynamically.